spk/sonarr/Makefile

```diff
@@ -16,7 +16,7 @@ DESCRIPTION_FRE = Sonarr est un PVR pour les utilisateurs de groupes de discussi
 DESCRIPTION_SPN = Sonarr es un PVR para los usuarios de grupos de noticias y torrents. Se puede controlar múltiples canales RSS para nuevos episodios de sus programas favoritos y se agarra, tipo y les cambia el nombre. También puede ser configurado para actualizar automáticamente la calidad de los archivos ya descargados si un formato de mejor calidad disponible.
 DISPLAY_NAME = Sonarr
 STARTABLE = yes
-CHANGELOG = "Enlarge the service start/stop timeout to 90 seconds."
+CHANGELOG = "1. Enlarge the service start/stop timeout to 90 seconds<br/>2. Avoid Sonarr downgrade on package update<br/>3. Upgrade to Sonarr 3.0.9.1549"
```

Comment:
The sonarr service should also be changed to use the alt temp directory like radarr/lidarr etc given that DSM 7 restrictions on /tmp and general future proofing

Reply:
Thanks for pointing this out. I'll update the PR to include the alternate temp directory.